### PR TITLE
Back to top arrow style updated

### PR DESCRIPTION
**Pull request by andysellick**
* SVG arrow added to fit with new design
* Font size also adjusted to fit new design

Before: 

screen shot 2017-05-22 at 08 57 19

After:

screen shot 2017-05-19 at 15 31 28

https://trello.com/c/J6YkyF1T/356-deployment-back-to-top-button-2

Now relates to https://github.com/alphagov/static/pull/1067

### CODE REVIEW

File: app/views/shared/_back_to_content_link.html.erb [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
-  },
-) %>
+<a class="back-to-content"
+  data-track-category="contentsClicked"
+  data-track-action="backToContentsLinkClicked"
+  data-track-label="contents"
+  data-module="track-click"
+  href="#contents">
+    <svg class="icon" width="14" height="17" viewBox="0 0 14 17" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g transform="translate(-885.000000, -1380.000000) translate(886.000000, 1380.000000) translate(0.000000, 1.000000)" stroke-width="2" stroke="currentColor" fill="none" fill-rule="evenodd"><polyline points="12 6.61354167 6.10565074 0.719192411 0.312351484 6.51249167"/><path d="M6,1.52415365 L6,15" stroke-linecap="square"/></g></svg>
+    Contents
```
Comment (fofr) [inline app/views/shared/_back_to_content_link.html.erb]: This should still be translated. It used to be `t('content_item.contents')`.

Comment (fofr) [inline app/views/shared/_back_to_content_link.html.erb]: You could keep the `link_to` method by doing this:

```erb
<%= link_to(params) do %>
 Link text
<% end %>
```

**Comment by fofr** [comment]
Would be good to show a before/after screenshot.

File: app/assets/images/arrow-up.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg width="14" height="17" viewBox="0 0 14 17" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g transform="translate(-885.000000, -1380.000000) translate(886.000000, 1380.000000) translate(0.000000, 1.000000)" stroke-width="2" stroke="#005EA5" fill="none" fill-rule="evenodd"><polyline points="12 6.61354167 6.10565074 0.719192411 0.312351484 6.51249167"/><path d="M6,1.52415365 L6,15" stroke-linecap="square"/></g></svg>
```
Comment (fofr) [inline app/assets/images/arrow-up.svg]: Is this file needed?

Comment (andysellick) [inline app/assets/images/arrow-up.svg]: Oops, good point

**Comment by fofr** [comment]
Tested this on old IE and it looks fine:
screen shot 2017-05-22 at 09 54 09

Tested on mobile and the arrow looks too big (font-size is reduced):
screen shot 2017-05-22 at 09 54 53

**Comment by fofr** [comment]
@nickcolley Can you add updated screenshots?

**Comment by nickcolley** [comment]
@fofr Andy is doing some additional work on this

**Comment by nickcolley** [comment]
I think we need to https://github.com/alphagov/government-frontend/blob/master/app/views/shared/_sidebar_contents_with_body.html.erb add this here too

**Comment by fofr** [comment]
I was hoping we could keep the design updates and the "adding to new formats" as separate PRs. One shouldn't block the other.

**Comment by nickcolley** [comment]
image

I think the nesting might be off, since it looks like we're missing the grids gutter padding

**Comment by andysellick** [comment]
Nesting should be fixed now.

**Comment by nickcolley** [comment]
I've noticed for some of the pages the contents link will spill over the end, I wonder if we can address this? It feels really odd at the moment.

screen shot 2017-05-25 at 14 47 14

**Comment by nickcolley** [comment]
Checking the live site, the issue I've raised above is already an issue, so we're going to ship these changes and try and fix the existing behaviour separately.

**Comment by andysellick** [comment]
Fix added to static... https://github.com/alphagov/static/pull/1067

File: app/views/content_items/document_collection.html.erb [hunk 3]
```diff
@@ -42,6 +46,9 @@
       </ol>
     <% end %>
   </div>
+  <div data-sticky-element class="govuk-sticky-element">
```
Comment (fofr) [inline app/views/content_items/document_collection.html.erb]: Use of the class name `govuk-sticky-element` hints that it's a component, which it isn't.

**Comment by fofr** [comment]
How does this look on right to left content?

**Comment by andysellick** [comment]
I can't find a page with right to left content where this is used. Any suggestions?

**Comment by andysellick** [comment]
Seems to be okay on RTL. Have added some more example links to the card.
